roles/edpm_ovn/meta/argument_specs.yml

```diff
@@ -105,35 +81,18 @@ argument_specs:
         default: 8000
         description: ''
         type: int
-      edpm_ovn_ovs_external_ids:
-        default:
-          hostname: '{{ ansible_nodename }}'
-          ovn-bridge: '{{ edpm_ovn_bridge }}'
-          ovn-bridge-mappings: '{{ edpm_ovn_bridge_mappings | join('','') }}'
-          ovn-chassis-mac-mappings: '{%- set chassis_mac_mappings = [] -%} {%- for physnet,
-            mac_prefix in edpm_ovn_chassis_mac_mapping_prefixes.items() -%} {{ chassis_mac_mappings.append([physnet,
-            mac_prefix | community.general.random_mac(seed=edpm_ovn_chassis_mac_mapping_seed)]
-            | join('':'')) }} {%- endfor -%} {{ chassis_mac_mappings | join('','') }}'
-          ovn-encap-ip: '{{ edpm_ovn_encap_ip }}'
-          ovn-encap-type: '{{ edpm_ovn_encap_type }}'
-          ovn-match-northd-version: true
-          ovn-monitor-all: true
-          ovn-ofctrl-wait-before-clear: '{{ edpm_ovn_ofctrl_wait_before_clear }}'
-          ovn-remote: '{% set db_addresses = [] %}{% for host in edpm_ovn_dbs %}{{ db_addresses.append([edpm_ovn_protocol,
-            host, edpm_ovn_sb_server_port] | join('':'')) }}{% endfor %}{{ db_addresses
-            | join('','') }}'
-          ovn-remote-probe-interval: '{{ edpm_ovn_remote_probe_interval }}'
-          rundir: /var/run/openvswitch
-        description: Sets external_id data from provided variables using Jinja templating
-        type: dict
+      ovn_match_northd_version:
```

Comment:
where these used? also naming convention differs - vs _

Reply:
Thanks for catching this, variable is now passed properly.